Makefile

```diff
@@ -52,6 +52,12 @@ pylint:
 	PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=$(DJANGO_SETTINGS_MODULE) pylint --load-plugins=pylint_django.checkers.migrations -d missing-docstring -d duplicate-code --module-naming-style=any  tcms/*/migrations/*
 	PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=$(DJANGO_SETTINGS_MODULE) pylint --load-plugins=pylint_django --load-plugins=kiwi_lint --load-plugins=pylint.extensions.docparams -d missing-docstring -d duplicate-code tcms/ tcms_settings_dir/
 
+.PHONY: pylint_site_packages
+pylint_site_packages:
+	if [ -d "$(PATH_TO_SITE_PACKAGES)" ]; then \
+		PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=tcms.settings.common pylint --load-plugins=kiwi_lint --disable=all --enable=avoid-generic-foreign-key $(PATH_TO_SITE_PACKAGES);\
```

Comment:
@brymut to avoid delays b/c of the pylint recursion issue I think we shold try to workaround it. If you loop over each 1st level dir under site-packages and then execute pylint for it it will probably not hit the recursion bug. Something like (untested) `find -type d -maxdepth 1`